modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -1005,8 +1005,7 @@ class CV_EXPORTS_W VideoWriter
     @param filename Name of the output video file.
     @param fourcc 4-character code of codec used to compress the frames. For example,
     VideoWriter::fourcc('P','I','M','1') is a MPEG-1 codec, VideoWriter::fourcc('M','J','P','G') is a
-    motion-jpeg codec etc. List of codes can be obtained at [Video Codecs by
-    FOURCC](http://www.fourcc.org/codecs.php) page. FFMPEG backend with MP4 container natively uses
+    motion-jpeg codec etc. FFMPEG backend with MP4 container natively uses
```

Comment:
I propose to replace the link with some other long-living resource like MSDN: https://docs.microsoft.com/en-us/windows/win32/medfound/video-fourccs